source/_integrations/monoprice.markdown

```diff
@@ -55,3 +55,67 @@ Restore a previously taken snapshot of one or more speakers. If `entity_id` is `
 | Service data attribute | Optional | Description |
 | ---------------------- | -------- | ----------- |
 | `entity_id` | yes | String or list of strings that point at `entity_id`s of zones.
+
+
+## Examples
+
+### Script
+
+```yaml
+# Take Snapshot of current state, turn on zones, select source.
+alias: Notify - Media Announment
+sequence:
+  - service: monoprice.snapshot
+    data: {}
```

Comment:
```suggestion
```

Reply:
Per your other suggestion.  I removed the script and made the requested changes.